local.defaults.properties

```diff
@@ -0,0 +1 @@
+MAPS_API_KEY=""
```

Comment:
Is this needed to avoid `null` values? I'm assuming so, but it's not clear if it's unused (I can't currently build the project).

Reply:
This file is needed for builds to pass if the machine doesn't contain the MAPS_API_KEY in the secrets properties file. I explain the reasoning for a default properties file here: https://github.com/google/secrets-gradle-plugin/issues/6